src/worker.js

```diff
@@ -8,11 +8,16 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
+const healthCheckEndpoint = '/';
+
 const attachMiddleware = (app, config) => {
   app.use(bodyParser.json(config.bodyParser));
 };
 
 const attachEndpoint = (app, config, callback) => {
+  app.get(healthCheckEndpoint, function (req, res) {
```

Comment:
prefer `(req, res) => { ... }` syntax here

(this is what's causing the CI job to fail)

https://travis-ci.org/airbnb/hypernova/jobs/245282577#L267